packages/terra-avatar/src/Avatar.scss

```diff
@@ -0,0 +1,56 @@
+@import '~terra-mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-color: var(--terra-avatar-background-color, rgb(222, 223, 224));
+    border: var(--terra-avatar-border, 0 none);
+    border-radius: 50%;
+    box-shadow: var(--terra-avatar-box-shadow);
+    color: var(--terra-avatar-color, rgb(155, 159, 161));
+    display: inline-flex;
+    height: var(--terra-avatar-size, 3.429rem);
+    justify-content: center;
+    line-height: 0;
```

Comment:
Why is this needed?

Reply:
Terra Base sets a line height that causes vertical spacing to be off centered.

Reply:
If this ends up being needed, can we include a comment about why?

Reply:
Good idea. Anyone else with thoughts around this?